### DISCUSSION TITLE
Is this a permanent fork?

### DISCUSSION THREAD

**Discussion by jpwilliams** [semantic-release-plus/semantic-release#215] [General]
Hi there! 👋

I'd been wanting to use `semantic-release` in a mono-repo so had just implemented commit filtering in a very, uh, haphazard manner. The `commitPaths` option here is _exactly_ what I'd want to see in the main `semantic-release` repo!

Are you intending for this to be a permanent, maintained fork? Or would the `commitPaths` feature hopefully be merged in to `semantic-release`?

**Comment by jpwilliams** [comment]
Ah - I have just seen https://github.com/semantic-release/semantic-release/issues/193#issuecomment-545118972 and #73.

**Reply by JoA-MoS** [reply]
Yes, I am maintaining and working with my new company to implement using it. 

I would like for both the commit filtering and the skip tagging functionality to be merged in but I have stopped trying, it may be worth one more attempt. 

I think the part that others are getting hung up on is the release ordering in a monorepo which is not really an issue and in my opinion shouldn't be something the this tool is even concerned about.
